Tests/IntegrationTests.swift

```diff
@@ -3,53 +3,76 @@ import XCTest
 import XpringKit
 
 extension Wallet {
-  /// A test wallet which contains funds.
-  public static let testWallet = Wallet(seed: "snYP7oArxKepd3GPDcrjMsJYiJeJB")!
+    /// A test wallet which contains funds.
+    public static let testWallet = Wallet(seed: "snYP7oArxKepd3GPDcrjMsJYiJeJB")!
 }
 
 extension String {
-  /// The URL of the remote gRPC service.
-  public static let remoteURL = "grpc.xpring.tech:80"
+    /// The URL of the remote gRPC service.
+    public static let remoteURL = "142.93.73.197:88"
```

Comment:
@keefertaylor To test this, you would need to get `grpc.xpring.tech:80` up and running on with the new Rpc_V1 protocols. Just FWI, I changed the legacy to my server and it failed. Just to note.